src/mem/cache/tags/compressed_tags.cc

```diff
@@ -104,7 +116,8 @@ CompressedTags::tagsInit()
 CacheBlk*
 CompressedTags::findVictim(Addr addr, const bool is_secure,
                            const std::size_t compressed_size,
-                           std::vector<CacheBlk*>& evict_blks)
+                           std::vector<CacheBlk*>& evict_blks,
+                           const uint64_t partition_id=0)
```

Comment:
Something must be done with the partition id here, as the implementation seems pretty trivial (preferred). Either that or assert that PPs are not used when this tag type is used.

Reply:
Added simple Partitioning logic to be used here in case policies are in place; Also added a warning that using PPs is not thoroughly tested in constructor for `SectorTags` (parent class of `CompressedTags`)